spec/features/current_location/user_views_nearby_alums_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+feature 'user views map of nearby alums', %{
+  As an authenticated user
+  I want to view the map by checking in
+  So that I can see if any alums are nearby
+} do
+  # Acceptance Criteria
+  # [] I must be a registered user
+  # [] I cannot view map unless I check in (updating my location)
+  # [] I should see myself on the map as well
+
+  let!(:user) { FactoryGirl.create(:user) }
+  let!(:current_location) { FactoryGirl.create(:current_location,
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.